projects/hangman/hangman.py

```diff
@@ -0,0 +1,53 @@
+"""
+    Script for making hangman game
+    :file: hangman.py
+    :platform: Windows 10
+    :synopsis:
+        This script is the parent file of hangman game.
+        All the functions that make up the whole code
+        are called here sequence wise. At the end, there
+        is built-in choice for user to either again
+        play the game or quit it.
+
+    :author:
+        Muhammad Faheem-ur-Rehman
+        email: faheemlasani1034@gmail.com
+"""
+
+import hangman_functions
+
+
+def main():
+    flag = True
+    key = True
+
+    #saving the game data in a separate dictionary
+    words_dict = {'COUNTRIES': ['GERMANY', 'UGANDA', 'FIJI', 'FINLAND', 'ARGENTINA', 'QATAR', 'EGYPT', 'AUSTRALIA'],
+                  'SCIENCE': ['VELOCITY', 'INERTIA', 'REACTION', 'FUSION', 'GRAVITY', 'FORCE', 'ELECTRONS',
+                              'RADIATION'],
+                  'FRUITS': ['MANGO', 'GUAVA', 'PEAR', 'APRICOT', 'MELON', 'GRAPES', 'ORANGE', 'FIG'],
+                  'PROFESSIONS': ['PHYSICIST', 'ASTRONOMER', 'BARTENDER', 'LIBRARIAN', 'GAMER', 'ACCOUNTANT', 'BUTLER',
+                                  'BIOLOGIST']}
```

Comment:
why not extend the arsenal of guess words for each category? ;)

Reply:
@M-Shan Four guess words added in each category